src/components/contextual/pages/vebal/LMVoting/GaugesTable.vue

```diff
@@ -90,16 +92,11 @@ const columns = ref<ColumnDefinition<VotingGaugeWithVotes>[]>([
   },
   {
     name: t('veBAL.liquidityMining.table.nextPeriodVotes'),
-    accessor(gauge) {
-      const normalizedVotes = scale(new BigNumber(gauge.votes), -18);
-      return fNum2(normalizedVotes.toString(), {
-        style: 'percent',
-        maximumFractionDigits: 2
-      });
-    },
+    acessor: 'id',
     align: 'right',
-    id: 'poolGaugeVotes',
-    sortKey: gauge => Number(gauge.votes),
+    id: 'nextPeriodVotes',
+    Cell: 'nextPeriodVotesCell',
+    sortKey: gauge => Number(gauge.votesNextWeek),
```

Comment:
`gauge.votesNextWeek` seems to throw a TS error, are you sure this property is available.